include/wx/dcsvg.h

```diff
@@ -225,14 +229,14 @@ class WXDLLIMPEXP_CORE wxSVGFileDCImpl : public wxDCImpl
    // their current values in wxDC.
    void DoStartNewGraphics();
 
-   wxFileOutputStream *m_outfile;
+   wxScopedPtr<wxFileOutputStream> m_outfile;
```

Comment:
Actually, why does this have to be a pointer at all? It looks like it could be just a member as we always have the file name to initialize it with in the ctor...